performance_regression_lwt_test.py

```diff
@@ -0,0 +1,36 @@
+from invoke.exceptions import UnexpectedExit, Failure
+
+from performance_regression_test import PerformanceRegressionTest
+from sdcm.utils.common import log_run_info, retrying
+
+
+class PerformanceRegressionLWTTest(PerformanceRegressionTest):
+
+    """
+    Test Scylla performance regression with cassandra-stress.
+    """
+
+    @log_run_info
+    @retrying(n=3, sleep_time=15, allowed_exceptions=(UnexpectedExit, Failure))  # retrying since SSH can fail with 255
+    def run_compaction_on_all_nodes(self):
+        for node in self.db_cluster.nodes:
+            node.run_nodetool("compact")
+
+    def _run_workload(self, param_name, subtype, keyspace_num=1):
+        cmd = self.params.get(param_name)
+        user_prefix = self.params.get('user_prefix')
+        subtype = f'{user_prefix}-{subtype}-{keyspace_num}'
+        test_name = f'LWTPerformanceRegressionTest.{subtype}'
+        self._workload(stress_cmd=cmd, stress_num=2, sub_type=subtype, test_name=test_name, keyspace_num=keyspace_num,
+                       save_stats=True, debug_message=f'Stress command {param_name} command: {cmd}')
+
+    def test_lwt_latency(self):
+        self.preload_data()
+        # self.alter_table_to_in_memory()
+        # restart needed to load data to in-memory store
+        # self.run_compaction_on_all_nodes()
+        self._run_workload('stress_cmd_lwt_ine', 'lwt-insert-not-exists')
+        self._run_workload('stress_cmd_lwt_uc', 'lwt-update-conditional')
```

Comment:
you need to wait for compactions to settle between each test, so they won't interfere with each other.

Reply:
Yap, will add it there, but I see another problem - we don't zero out and/or locolize Prometheus counters that we use for latancies and throughput in between ._workload runs.
So these counters are interfere in between runs.